service-registry-client/src/main/java/no/difi/meldingsutveksling/serviceregistry/externalmodel/BrregPostadresse.java

```diff
@@ -0,0 +1,22 @@
+package no.difi.meldingsutveksling.serviceregistry.externalmodel;
+
+import lombok.Data;
+
+@Data
+public class BrregPostadresse {
+
+    private String adresse;
+    private String postnummer;
+    private String poststed;
+    private String land;
+
+    public BrregPostadresse() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1186)